tgstation.dme

```diff
@@ -8326,6 +8326,7 @@
 #include "modular_zubbers\modules\customization\modules\language\_language_holder.dm"
 #include "modular_zubbers\modules\customization\modules\language\nekomimetic.dm"
 #include "modular_zubbers\modules\customization\modules\language\piratespeak.dm"
+#include "modular_zubbers\modules\customization\modules\mob\dead\taur_fatnaga.dm"
```

Comment:
change this for "modular_zubbers/modules/customization/modules/mob/dead/taur_types.dm"

Reply:
I accadently hit resolve so ignore that. How do I edit it within the browser.